app/javascript/packages/document-capture/context/upload.jsx

```diff
@@ -50,6 +50,7 @@ UploadContext.displayName = 'UploadContext';
  * @prop {false} success Whether request was successful.
  * @prop {UploadFieldError[]=} errors Error messages.
  * @prop {string=} redirect URL to which user should be redirected.
+ * @prop {Number=} remaining_attempts Number of remaining doc capture attempts for user.
```

Comment:
Minor: This should be lowercase, since the capitalized `Number` refers to a different thing.

See: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

```suggestion
 * @prop {number=} remaining_attempts Number of remaining doc capture attempts for user.
```

Reply:
Damned, I even looked that up and was surprised to find it had a primitive vs. Object number. I thought about fixing it but apparently never did.